src/frontend/flight/handler.rs

```diff
@@ -61,8 +62,14 @@ impl SeafowlFlightHandler {
             self.context.clone()
         };
 
-        let plan = ctx.plan_query(query).await?;
-        let batch_stream = ctx.execute_stream(plan).await?;
+        let plan = ctx
+            .plan_query(query)
+            .await
+            .inspect_err(|err| debug!("Error planning query id {query_id}: {err}"))?;
+        let batch_stream = ctx
+            .execute_stream(plan)
+            .await
+            .inspect_err(|err| debug!("Error executing query id {query_id}: {err}"))?;
```

Comment:
These could be INFO I think (since we can't turn on debug logging on the fly without redeploying Seafowl)